tests/fl/monai_algo/test_fl_monai_algo.py

```
@@ -179,6 +179,36 @@
+    for obj in gc.get_objects():
+        if not isinstance(obj, Engine):
+            continue
+        try:
+            url = obj.url
+            db = url.database if url.get_backend_name() == "sqlite" else None
+            # the test backends are all files named ``mlflow*.db``; match those only so
+            # unrelated (e.g. in-memory) sqlite engines in the process are left untouched.
+            if db and os.path.basename(db).startswith("mlflow"):
+                obj.dispose()
+        except Exception:
+            pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)